README.md

```diff
@@ -1 +1,45 @@
-# create_engineers_website
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# creative engineers website:  mobile and desktop version 
+> building the first 2 pages (Home and About pages) of a fantasy company that i dreamed to establish before  for mobile and Dektop devices.
+that was done by choosing [This Template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) that is actually designed by this creative graphic desginer Guy  [Cindy Shin in Behance](https://www.behance.net/adagio07).
+
+
+## Built With
+
+- html , css and javascript
+
+
+## Live Demo 
+
+[Live Demo Link](https://mohamed20163858.github.io/create_engineers_website/)
+
+
+## Getting Started
+You can download the page by clicking in the code green button and select Download ZIP or if you are a developer 
+you can get the https link or ssh link just copy it and use your favourite command line git interface and write this `git clone https://github.com/mohamed20163858/create_engineers_website.git` if you use https or write 
+`git@github.com:mohamed20163858/create_engineers_website.git` if you use ssh and then after installition you can run the application by just clicking on index.html inside my-portfolio directory 
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@mohamed20163858](https://github.com/mohamed20163858)
+- Twitter: [@mohamed20163858](https://twitter.com/mohamed20163858)
+- LinkedIn: [mohamed-mohsen-685a4995](https://www.linkedin.com/in/mohamed-mohsen-685a4995/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
```

Comment:
your readme should contain an `Acknowledgements` section to give credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
@alexander16108 ok i will do that